tests/test_logic.py

```diff
@@ -948,7 +948,7 @@ def test_Controller_on_seen_success(homedir, mocker, session_maker):
 def test_Controller_on_seen_failure(homedir, mocker, session_maker):
     co = Controller("http://localhost", mocker.MagicMock(), session_maker, homedir, None)
     debug_logger = mocker.patch("securedrop_client.logic.logger.debug")
-    error = Exception("errorororr")
+    error = Exception("error error")
```

Comment:
notable

Reply:
This looks intentional. One `error` is enough I think :slightly_smiling_face: 

(I _knew_ when I saw it that it was you @creviera :smile: :heart:)